gke-deploy/core/resource/resource.go

```diff
@@ -148,28 +148,28 @@ func ParseConfigs(ctx context.Context, configs string, oss services.OSService, r
 // If any lines in a resource object's string representation contain a key in
 // lineComments, the corresponding value will be added as a comment at the end of
 // the line.
```

Comment:
Maybe just add a sentence saying what is being returned?

Reply:
Done.